src/view/selection.js

```diff
@@ -314,6 +315,29 @@ export default class Selection {
 	}
 
 	/**
+	 * Returns {@link engine.view.EditableElement EditableElement} instance that contains this selection.
+	 *
+	 * @returns {engine.view.EditableElement|null} Returns closest EditableElement or null if none is found.
+	 */
+	getEditableElement() {
+		if ( this.rangeCount ) {
+			let editable = this.getFirstPosition().parent;
```

Comment:
We may introduce `Position.getEditableElement()` and use it here: `this.getFirstPosition().getEditableElement()`.